Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+// vim: et:ts=2:sw=2
+// This Jenkinsfile is intended to be used with a companion jenkins-job-builder
+// definition. Aside from the credentials it requires which are accessed in the
+// 'environment' section of this pipeline, it requires the following parameters:
+//   AWS_DOMAIN
+//   AWS_PROFILE
+//   AWS_REGION
+//   CLUSTER_USER
+// It may also provide these optional parameters to override the framework's
+// defaults:
+//   ROOK_IMAGE
+//   CEPH_IMAGE
+pipeline {
+  agent { node { label "rhel-8-large" }}
+  environment {
+    AWS_SHARED_CREDENTIALS_FILE = """${sh(
+      returnStdout: true,
+      script: 'echo $WORKSPACE/.aws/credentials'
+      )}"""
+    AWS_CONFIG_FILE = """${sh(
```

Comment:
WORKSPACE variable is directly available from pipeliony, Why not just AWS_CONFIG_FILE = "${WORKSPACE}/.aws/config" , the same above

Reply:
I thought that didn't work with declarative pipelines? I could be mistaken here. Happy to change if that's the case.

Reply:
I think it should work also in declarative pipeline, maybe it's just ${env.WORKSPACE}